qiskit/circuit/quantumcircuit.py

```diff
@@ -2265,17 +2265,98 @@ def reset(self, qubit: QubitSpecifier) -> InstructionSet:
         return self.append(Reset(), [qubit], [])
 
     def measure(self, qubit: QubitSpecifier, cbit: ClbitSpecifier) -> InstructionSet:
-        """Measure quantum bit into classical bit (tuples).
+        """Measure quantum bit (qubit) into classical bit (cbit).
+
+        When a qubit is measured, its state collapses to a classical bit and copied to a
+        classical wire.
 
         Args:
-            qubit: qubit to measure.
-            cbit: classical bit to place the measurement in.
+            qubit: qubit/s to measure.
+            cbit: classical bit/s to place the measurement in.
 
         Returns:
             qiskit.circuit.InstructionSet: handle to the added instructions.
 
         Raises:
             CircuitError: if arguments have bad format.
+
+        Examples:
+            In this example, a qubit is measured and the result of that measurement is stored in the
+            classical bit (usually expressed in diagrams as a double line):
+
+            .. code-block::
+
+               from qiskit import QuantumCircuit
+               circuit = QuantumCircuit(1,1)
+               circuit.h(0)
+               circuit.measure(0, 0)
+               circuit.draw()
+
+
+            .. parsed-literal::
+
+                      ┌───┐┌─┐
+                   q: ┤ H ├┤M├
+                      └───┘└╥┘
+                 c: 1/══════╩═
+                            0
+
+            It is possible to call ``measure`` with lists of qubits and cbits of the same size:
+
+            .. code-block::
+
+               circuit = QuantumCircuit(2,2)
+               circuit.measure([0,1], [0,1])  # same as "circuit.measure(0,0); circuit.measure(1,1);"
+
+
+            .. parsed-literal::
+
+                      ┌─┐
+                 q_0: ┤M├───
+                      └╥┘┌─┐
+                 q_1: ─╫─┤M├
+                       ║ └╥┘
+                 c: 2/═╩══╩═
+                       0  1
+
+            It is also possible to do a one-to-many readout:
+
+            .. code-block::
+
+               circuit = QuantumCircuit(2,2)
+               circuit.measure(0, [0,1])  # same as "circuit.measure(0,0); circuit.measure(0,1);"
+
+
+            .. parsed-literal::
+
+                      ┌─┐┌─┐
+                 q_0: ┤M├┤M├
+                      └╥┘└╥┘
+                 q_1: ─╫──╫─
+                       ║  ║
+                 c: 2/═╩══╩═
+                       0  1
```

Comment:
This is technically possible, but a bit useless haha.

Reply:
I added because I think it is in the specification. Do you think would be better not to have it?

Reply:
I don't feel particularly strongly, though I'd probably at least mention that there's little reason to do it.  Similarly, I think it's technically possible to use the broadcast to measure several qubits into the same classical bit, but that's even more useless, and I definitely wouldn't mention it.

Reply:
FWIK many-to-one is explicitly prevented.

Reply:
Thinking over it, it is getting long and there is no real benefit. So I removed that case in 7bc4849f886061e04aed136e7e81de47b0e79423

Reply:
Thinking over it, it is getting long and there is no real benefit. So I removed that case in 7bc4849f886061e04aed136e7e81de47b0e79423